monocle/overseer.py

```diff
@@ -436,7 +437,10 @@ def get_start_point(self):
                         await self.coroutine_semaphore.acquire()
                         LOOP.create_task(self.try_point(mystery_point))
                     except IndexError:
-                        self.mysteries = SPAWNS.get_mysteries()
+                        if next_mystery_reload < time.time():
+                            self.mysteries = SPAWNS.get_mysteries()
+                            next_mystery_reload = time.time() + min(conf.GIVE_UP_UNKNOWN, 90)
```

Comment:
What is the relevance of `GIVE_UP_UNKNOWN` here? Seems like this should either be a constant or a new config option.